fission/src/ui/panels/DeveloperToolPanel.tsx

```diff
@@ -172,7 +250,17 @@ const DeveloperToolPanel: React.FC<PanelPropsImpl> = ({ panelId }) => {
                             {selectedKey ? (
                                 <>
                                     {/* strip off the prefix here */}
-                                    <div className="font-bold text-sm mb-2">{selectedKey.replace(/^devtool:/, "")}</div>
+                                    {selectedKey === "devtool:scoring_zones" ? (
+                                        LabelWithTooltip(
+                                            "scoring_zones",
+                                            `Add and cache scoring zones. \n Example:\n[\n  {\n    \"name\": \"Red Zone\",\n    \"alliance\": \"red\",\n    \"parentNode\": \"root\",\n    \"points\": 5,\n    \"destroyGamepiece\": false,\n    \"persistentPoints\": true,\n    \"deltaTransformation\": [1,0,0,0,0,1,0,0,0,0,1,0,0,0,0,1]\n  }\n]`,
```

Comment:
You don't need the `\` to escape the`"`, since the it's already a format string which uses the grave character as a delimiters instead. However, you don't need a format string for this (since you're not formatting any variables) and should probably switch to single quotes (`'`) instead.